spec/mailers/previews/candidate_mailer_preview.rb

```diff
@@ -392,6 +392,14 @@ def reference_received
     CandidateMailer.reference_received(reference)
   end
 
+  def reference_received_after_selection
+    application_form_with_selected_references = FactoryBot.create(:application_form)
```

Comment:
We need to use FactoryBot.build_stubbed here please  or it actually creates objects in the qa database everytime someone views it